presto/presto_test.go

```diff
@@ -275,6 +275,52 @@ func TestQueryFailure(t *testing.T) {
 	}
 }
 
+func TestExec(t *testing.T) {
+	c := &Config{
+		PrestoURI:         "http://foobar@localhost:8080",
+		SessionProperties: map[string]string{"query_priority": "1"},
+	}
+	dsn, err := c.FormatDSN()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	db, err := sql.Open("presto", dsn)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer db.Close()
+
+	rows, err := db.Query(`SELECT * FROM "presto.execution:name=queryexecution"`)
+	expected := "Schema must be specified when session schema is not set"
+	if err == nil || !strings.Contains(err.Error(), expected) {
+		t.Fatalf("Expected to fail to execute query with error: %v, got: %v", expected, err)
+	}
+
+	result, err := db.Exec("USE jmx.current")
```

Comment:
Why is `TestUnsupportedExec` below not failing with your change?

Reply:
I totally missed that! It looks like that test was invalid as it only checked if the err is nil. I removed this one and fixed the following one for unsupported trx by adding an assert for a specific error message.